### PR TITLE
feat: add option to skip ci run

### PR DESCRIPTION
**Pull request by nenadfilipovic**
PR summary:

- Added  feature to skip ci run
- Added configuration to disable or enable prompt
- Added chinese translation
- Update documentation with prompt config

Fix: #212 

@vivaxy please check if chinese translation is correct I have used google translate

### CODE REVIEW

**Comment by vivaxy** [comment]
@nenadfilipovic Cool PR!
I think we should set `promptCi` as `false` by default to preserve current user experience.

`skip ci` feature can work in many ways. Ref: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/. Should we provide configurations to:
1. Set `[skip ci]` position. e.g. head or tail in header section.
2. Set different keyword? e.g. `[skip ci]`, `[ci skip]`, e.t.c.

**Comment by nenadfilipovic** [comment]
Ok I have changed it to be false by default.

To answer other 2 questions:

1. I don't have strong opinion on where should `[skip ci]` be, I usually put it at the end of the header
2. By looking here [https://www.scivision.dev/skip-ci-builds-git-commit/](https://www.scivision.dev/skip-ci-builds-git-commit/) I came to conclusion that `[skip ci]` is universally accepted form and I think we should use it

**Review by vivaxy — APPROVED** [review]

**Comment by vivaxy** [comment]
@yi-Xu-0100 Any more questions?

**Comment by yi-Xu-0100** [comment]
👋 @nenadfilipovic, thanks for your great contribution! ❤ I think it is a clear way to prompt the condition about the skip CI. After reviewing the realization about the prompt, I have a little advice about the format of the prompt.

@vivaxy May we can make the prompt to have choices with two options? The first one is `No`, and it will add none character into the placeholder and will be the default choice. The second one is `Yes`, and it will replace the character of the placeholder with `yes`. The format will be like the emoji prompt, and it can be picked with the mouse easier. 🧐

**Comment by nenadfilipovic** [comment]
@vivaxy @yi-Xu-0100 do you think PR is good?

**Comment by yi-Xu-0100** [comment]
@vivaxy, I think a little discussion is needed here about my reviews. 🧐

 I had made a beta extension for the current pr to be used temporarily if you require it- <https://github.com/vivaxy/vscode-conventional-commits/pull/213#issuecomment-985531262>. 😁@nenadfilipovic

**Comment by vivaxy** [comment]
Look Good To Me.

> For the above optional skip CI prompt, maybe it should prompt after choose the emoji.

Second thought, why is that? `[skip ci]` is appended after `subject`. Why is it not prompt after `subject`?

**Review by vivaxy — APPROVED** [review]

**Comment by yi-Xu-0100** [comment]
@vivaxy I think the selection can be simply completed by a mouse so that it should be prompt after emoji, and after that, it all will be keyboard things (even we can use `enter` and arrow keys to complete selection).

By the way, what is your opinion about the configuration variable naming and translation I suggested above?

**Comment by nenadfilipovic** [comment]
I can't say anything about Chinese translation because I don't speak it. We can also use promptSkipCI no problem about that.

File: package.nls.zh-cn.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "extension.configuration.promptBody.markdownDescription": "是否需要填写 `body`。",
   "extension.configuration.promptFooter.markdownDescription": "是否需要填写 `footer`。",
   "extension.configuration.promptScopes.markdownDescription": "是否需要填写 `scope`。",
+  "extension.configuration.promptCi.markdownDescription": "控制扩展是否应提示跳过 CI 运行。",
```
[the remark this answers is not in the document]
Comment (vivaxy) [reply package.nls.zh-cn.json:14]: "是否需要提示“跳过 CI 工作流”。" is better.

**Comment by vivaxy** [comment]
Thanks @yi-Xu-0100 . It's a thoughtfully review.

**Comment by yi-Xu-0100** [comment]
@vivaxy I have pushed the changes to this PR. If there is no other problem, I will squash and merge this PR and publish it next Friday. 🧐

@nenadfilipovic Thanks for your great PR! ❤

**Comment by nenadfilipovic** [comment]
Good job, very nice experience working with you @vivaxy @yi-Xu-0100 👋

**Pull request merged**